pasqal_cloud/endpoints.py

```diff
@@ -16,29 +16,93 @@
 
 from dataclasses import dataclass
 
+import typing
```

Comment:
Can you import all types up here so we don't have `typing.value` in the code base